### PR TITLE
fix: remove access column header

### LINKED ISSUE #2974: Remove the Access column and right align the access buttons
## Current behavior

image

The Access column displays multiple buttons to access the workspace.

## Expected behavior

image

Remove the Access column and display the buttons aligned right in the table row.

Additionally, we can also remove the table column border between Resource and Agent when there are no agents.

### PR DESCRIPTION
**Pull request by AbhineetJain**
This PR removes the Access column header on the workspace page and aligns the access buttons to the right of the table row.

Additionally, we also remove the table column border when the resource does not have an agent.

## Subtasks

- [x] Remove Access table heading
- [x] Right align access buttons
- [x] Remove border from table column when no agents are present

## Screenshot
Screen Shot 2022-07-13 at 12 23 26 PM

Fixes #2974

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
@AbhineetJain one small thing, can you make the `paddingLeft` of the "Agent" column the same as the "Resource" column?

File: site/src/components/Resources/Resources.tsx [hunk 1]
```diff
@@ -184,5 +183,6 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     gap: theme.spacing(0.5),
     flexWrap: "wrap",
+    justifyContent: "right",
```
Comment (BrunoQuaresma) [inline site/src/components/Resources/Resources.tsx:186]: I think you should use `flex-end`. I think "right" is not well supported. This should be the reason for Chromatic getting different styles.

**Review by BrunoQuaresma — APPROVED** [review]
LGTM, just minor stuff before the merge.

File: site/src/components/Resources/Resources.tsx [hunk 2]
```diff
@@ -80,7 +79,7 @@ export const Resources: FC<ResourcesProps> = ({
                 if (!agent) {
                   return (
                     <TableRow key={`${resource.id}-${agentIndex}`}>
-                      <TableCell className={styles.resourceNameCell}>
```
[the remark this answers is not in the document]
Comment (AbhineetJain) [reply site/src/components/Resources/Resources.tsx:83]: We're using it to show the border when agents exist. Removed it from the case when no agents exist.

**Pull request merged**